packages/runtime/container-runtime/src/gc/gcConfigs.ts

```diff
@@ -24,8 +24,10 @@ import {
 	runSessionExpiryKey,
 	runSweepKey,
 	stableGCVersion,
+	throwOnTombstoneLoadKey,
+	throwOnTombstoneUsageKey,
 } from "./gcDefinitions";
-import { getGCVersion, shouldAllowGcSweep } from "./gcHelpers";
+import { getGCVersion, shouldAllowGcSweep, shouldAllowGcTombstoneEnforcement } from "./gcHelpers";
```

Comment:
You can probably remove `shouldAllowGcTombstoneEnforcement` from `gc/index.ts`.  Maybe the two `Key`s above too?

Reply:
Good catch. On removing that, I realized BlobManager has the logic to initialize `tombstoneFailOnLoad` too.